### PR TITLE
remove redundant semicolons

### PR DESCRIPTION
**Pull request by shengyu7697**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [x] The PR is proposed to proper branch
- [ ] There is reference to original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [x] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

File: modules/gapi/include/opencv2/gapi/own/types.hpp [hunk 1]
```diff
@@ -47,9 +47,9 @@ class Rect
 {
 public:
     Rect() = default;
-    Rect(int _x, int _y, int _width, int _height) : x(_x), y(_y),   width(_width),  height(_height)  {};
+    Rect(int _x, int _y, int _width, int _height) : x(_x), y(_y),   width(_width),  height(_height)  {}
```
Comment (TolyaTalamanov) [inline modules/gapi/include/opencv2/gapi/own/types.hpp:50]: There is an extra space: `_ _ { }`

File: modules/gapi/include/opencv2/gapi/own/types.hpp [hunk 2]
```diff
@@ -104,9 +104,9 @@ class Size
 {
 public:
     Size() = default;
-    Size(int _width, int _height) : width(_width),  height(_height)  {};
+    Size(int _width, int _height) : width(_width),  height(_height)  {}
```
Comment (TolyaTalamanov) [inline modules/gapi/include/opencv2/gapi/own/types.hpp:107]: Extra space

File: modules/gapi/include/opencv2/gapi/own/types.hpp [hunk 3]
```diff
@@ -27,7 +27,7 @@ class Point
 {
 public:
     Point() = default;
-    Point(int _x, int _y) : x(_x),  y(_y)  {};
+    Point(int _x, int _y) : x(_x),  y(_y)  {}
```
Comment (TolyaTalamanov) [inline modules/gapi/include/opencv2/gapi/own/types.hpp:30]: Extra space

**Review by TolyaTalamanov — APPROVED** [review]
LGTM, thanks!

Could I ask you also remove extract space before `{}` ?

**Comment by shengyu7697** [comment]
Ok, I removed extra spaces

**Review by TolyaTalamanov — APPROVED** [review]
Thanks! 👍

**Pull request merged**